catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardSharingPolicyPlugin.java

```diff
@@ -38,30 +38,31 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.apache.commons.lang3.StringUtils;
-import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceMetacardImpl;
+import org.codice.ddf.catalog.ui.metacard.sharing.ShareableMetacardImpl;
 
-public class WorkspaceSharingPolicyPlugin implements PolicyPlugin {
+public class ShareableMetacardSharingPolicyPlugin implements PolicyPlugin {
 
-  private static Map<String, Set<String>> getWorkspaceSecurityTag() {
-    return ImmutableMap.of(Constants.IS_WORKSPACE, ImmutableSet.of(Constants.IS_WORKSPACE));
+  private static Map<String, Set<String>> getSharingSecurityTag() {
+    return ImmutableMap.of(Constants.IS_SHAREABLE, ImmutableSet.of(Constants.IS_SHAREABLE));
   }
 
   private static Map<String, Set<String>> getOwnerPermission(String owner) {
     return ImmutableMap.of(Core.METACARD_OWNER, ImmutableSet.of(owner));
   }
 
-  private Map<String, Set<String>> getOwnerPermission(WorkspaceMetacardImpl workspace) {
-    return Optional.of(workspace)
-        .map(WorkspaceMetacardImpl::getOwner)
+  private Map<String, Set<String>> getOwnerPermission(ShareableMetacardImpl shareableMetacard) {
+    return Optional.of(shareableMetacard)
+        .map(ShareableMetacardImpl::getOwner)
         .filter(StringUtils::isNotEmpty)
-        .map(WorkspaceSharingPolicyPlugin::getOwnerPermission)
+        .map(ShareableMetacardSharingPolicyPlugin::getOwnerPermission)
         .orElseGet(ImmutableMap::of);
   }
 
-  private Map<String, Set<String>> getIndividualPermission(WorkspaceMetacardImpl workspace) {
+  private Map<String, Set<String>> getIndividualPermission(
+      ShareableMetacardImpl shareableMetacard) {
     return ImmutableMap.of(
         Security.ACCESS_INDIVIDUALS,
-        Optional.of(workspace)
+        Optional.of(shareableMetacard)
             .map(w -> w.getAttribute(Security.ACCESS_INDIVIDUALS))
             .map(Attribute::getValues)
```

Comment:
✏️ Can now be replaced with: 
```
.map(s -> s.getAccessIndividuals())
```
which can be reduced down to a method reference.